task_8/toma.lobanova/homework8/src/main/java/ua/od/hillel/human/Human.java

```diff
@@ -0,0 +1,80 @@
+package ua.od.hillel.human;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+public class Human {
+    String name = "unspecified";
```

Comment:
It would be a great idea to extract "unspecified" word into a constant and use it in all the "unspecified" places